src/components/FilterMenu/FilterMenu.js

```diff
@@ -301,12 +308,25 @@ const FilterMenu = ({
     }
   }
 
+  const handleAutoRefresh = itemId => {
+    const refreshValue = autoRefresh === itemId ? '' : AUTO_REFRESH
+    setAutoRefresh(prev => refreshValue)
+  }
+
   useEffect(() => {
     return () => {
       dispatch(removeFilters())
     }
   }, [params.pageTab, params.projectName, page, params.jobName, dispatch])
 
+  useEffect(() => {
+    if (tab === MONITOR_JOBS_TAB && autoRefresh === AUTO_REFRESH && !hidden) {
```

Comment:
don't use comparison for a tab. Add a new prop instead

Reply:
done.